pages/instructorGradebook/instructorGradebook.js

```diff
@@ -10,10 +10,12 @@ var sqlLoader = require('@prairielearn/prairielib/sql-loader');
 
 var sql = sqlLoader.loadSqlEquiv(__filename);
 
+const sanitizeStringForUrl = (str) => str.replace(/[^a-zA-Z0-9]/g, '_');
```

Comment:
It would be nice to retain `-` instead of making it an underscore (ex: "CS 296-25").  This also more consistently maintains the existing behavior: `str.replace(/[^a-zA-Z0-9-]/g, '_');`

Reply:
@nwalters512 I'm on board with Wade's suggestion to include hyphens in the whitelist, but if you make this change then we should probably also change the equivalent whitelist in `assessment.js` for consistency.